### PR TITLE
fixed cb

### PR DESCRIPTION
**Pull request by jmazanec15**
*Issue #, if available:*
#62 
*Description of changes:*
Fix KNNStatsRequest call in Circuit Breaker and add an Integ Test case to validate CB.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreaker.java [hunk 1]
```diff
@@ -35,6 +36,7 @@
  */
 public class KNNCircuitBreaker {
     private static Logger logger = LogManager.getLogger(KNNCircuitBreaker.class);
+    public static int CB_TIME_INTERVAL = 2*60;
```
Comment (vamshin) [inline src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreaker.java:40]: Nit: Add comment to mention seconds?

Comment (jmazanec15) [reply src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreaker.java:40]: Will add

File: src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreakerIT.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    public void testCbTripped() throws Exception {
+        tripCb();
+    }
+
+    public void testCbUntrips() throws Exception {
+        updateClusterSettings("knn.circuit_breaker.triggered", "true");
+        //TODO: Attempt to find a better way to trigger runnable than waiting 2 minutes for it to finish
+        Thread.sleep(CB_TIME_INTERVAL*1000);
+        assertFalse(isCbTripped());
```
Comment (vamshin) [inline src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreakerIT.java]: Do we still need to sleep here?

Comment (vamshin) [inline src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreakerIT.java]: Looks like BackoffPolicy takes care of waiting. We can remove these assertions?

Comment (jmazanec15) [reply src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreakerIT.java]: Good catch. I must have forgotten to remove that

Comment (jmazanec15) [reply src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNCircuitBreakerIT.java]: Yes. Will update.

**Review by vamshin — APPROVED** [review]
LGTM!

**Pull request merged**